auth.php

```diff
@@ -37,6 +37,14 @@
 $pagetitle = $SITE->shortname.': '.get_string('mfa', 'tool_mfa');
 $PAGE->set_title($pagetitle);
 
+// The only page action allowed here is a logout if it was requested.
+$logout = optional_param('logout', false, PARAM_BOOL);
+if ($logout) {
+    \tool_mfa\manager::mfa_logout();
+    // Now redirect home.
+    redirect(new \moodle_url($CFG->wwwroot));
```

Comment:
Would it be possible to redirect back to the original wants url? so that after the second loop through you still end up in the right place

Reply:
Ah thats a nice feature.